packages/ui/src/components/TreeView/TreeView.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Card from "../Card";
+
+interface TreeViewItemProps {
+  noBranch?: boolean;
+  children: React.ReactNode;
+}
+
+const Item = ({ noBranch, children, ...rest }: TreeViewItemProps) => (
+  <div
+    className="tree-view-item"
+    data-h2-margin-left="base(x2)"
+    data-h2-padding-left="base(x1)"
+    data-h2-padding-top="base(x1)"
+    data-h2-border-left="base:selectors[:not(:last-child)](1px solid gray.light) base:selectors[:last-child:before](1px solid gray.light)"
+    data-h2-content="base:selectors[::before]('')"
+    data-h2-display="base:selectors[::before](block)"
+    data-h2-position="base(relative) base:selectors[::before](absolute)"
+    data-h2-border-bottom="base:selectors[::before](1px solid gray.light)"
+    data-h2-left="base:selectors[::before](0)"
+    data-h2-top="base:selectors[::before](0)"
+    data-h2-height="base:selectors[::before](calc(50% + x.5))"
+    data-h2-z-index="base(0) base:selectors[:last-child:before](-1)"
+    {...(noBranch ? {} : { "data-h2-width": "base:selectors[::before](x1)" })}
+    {...rest}
+  >
+    {children}
+  </div>
+);
+
+interface TreeViewRootProps {
+  title: string;
+  subtitle?: React.ReactNode;
+  children: React.ReactNode;
+}
+
+const Root = ({ title, subtitle, children, ...rest }: TreeViewRootProps) => (
+  <div {...rest}>
+    {/* TODO: Should the card be kept in the tree view component or manually added? */}
+    <Card title={title} color="white" bold>
```

Comment:
Just my opinion, but I'd prefer to just pass this in with the children. That way we don't **_have_** to use a `Card` and have more control over the content here.

If we are sticking with a `Card` by default here could we at least drill some `CardProps` here to have more control over it and remove the TO DO?

```tsx
interface TreeViewRootProps {
  title: string;
  subtitle?: React.ReactNode;
  children: React.ReactNode;
  cardProps?: Omit<CardProps, "title">
}

<Card title={title} color="white" bold {...cardProps}>
```

Reply:
Sounds good I removed the Card so we can have more control over what gets added [bebc221](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6173/commits/bebc22173522ed79d6fb405a6a90d58aa28ca170)